src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -54,14 +70,33 @@ ReportCmcStatus::writeReport( const xrt_core::device* /*_pDevice*/,
 {
   boost::property_tree::ptree empty_ptree;
 
-  _output << "CMC status\n";
+  _output << "CMC\n";
   auto& cmc = _pt.get_child("cmc");
   if(cmc.empty()) {
     _output << "  Information unavailable" << std::endl;
     return;
   }
-  uint32_t err_code = cmc.get<uint32_t>("heartbeat_err_code");
-  _output << boost::format("  %-22s : 0x%x %s\n") % "Status" % err_code % cmc.get<std::string>("status");
-  if (err_code)
-    _output << boost::format("  %-22s : %s sec\n\n") % "err time" % cmc.get<std::string>("heartbeat_err_time");
+  try {
+    uint32_t err_code = cmc.get<uint32_t>("heartbeat_err_code");
+    _output << boost::format("  %-22s : 0x%x %s\n") % "Status" % err_code % cmc.get<std::string>("status");
+    if (err_code)
+      _output << boost::format("  %-22s : %s sec\n\n") % "err time" % cmc.get<std::string>("heartbeat_err_time");
+  } catch(...) {}
+  try {
+    _output << boost::format("  %-22s :\n") % "Runtime clock scaling feature";
+    _output << boost::format("  \t%-22s : %s\n") % "Supported" % cmc.get<std::string>("scaling_support");
+    _output << boost::format("  \t%-22s : %s\n") % "Enabled" % cmc.get<std::string>("scaling_enabled");
+    _output << boost::format("  \t%-22s:\n") % "Critical threshold (clock shutdown) limits";
+    _output << boost::format("  \t\t%-22s : %s\n") % "Power" % cmc.get<std::string>("scaling_critical_power_threshold");
+    _output << boost::format("  \t\t%-22s : %s\n") % "Temperature" % cmc.get<std::string>("scaling_critical_temp_threshold");
+    _output << boost::format("  \t%-22s:\n") % "Throttling threshold limits";
+    _output << boost::format("  \t\t%-22s : %s\n") % "Power" % cmc.get<std::string>("scaling_threshold_power_limit");

```

Comment:
```suggestion
    _output << boost::format("  \t\t%-22s : %s Watts\n") % "Power" % cmc.get<std::string>("scaling_threshold_power_limit");

```

Please add watts and C to the outputs